frontend/src/app/modules/boards/drag-and-drop/drag-and-drop.service.ts

```diff
@@ -124,14 +129,6 @@ export class DragAndDropService implements OnDestroy {
       el.dataset.sourceIndex = DragAndDropHelpers.findIndex(el).toString();
     });
 
-    this.drake.on('over', (el:HTMLElement, container:HTMLElement) => {
```

Comment:
Is there a specific reason the drop-one effect was removed? I somehow missed the visual effect. I also didn't realize that I couldn't drag a WP into a list just because there was no ghost image.

Reply:
I searched for references and determined that this doesn't do anything, but apparently I was wrong. I will revert that

Reply:
The styles come from [here](https://github.com/opf/openproject/blob/dev/app/assets/stylesheets/openproject/_generic.sass#L90).